.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,53 @@
+name: Build and Deploy Doc Site
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches:
+      - master
+      - issue-9-doc-as-readable-site
```

Comment:
`issue-9-doc-as-readable-site` is this a default branch?

Reply:
`issue-9-doc-as-readable-site` this is not a default branch. this has been added for testing purpose, and as mentioned in the PR description, needs to be removed after approval.
```
Once this PR is approved, we will need to remove the additional [branch name](https://github.com/CraftsmenLtd/BloodConnect/compare/issue-9-doc-as-readable-site?expand=1#diff-87e860af4b9db6c503ed680b6edb6333c6f8d7659f7d1a779a2487466bbc50f6R8) from the action yml file and create another commit. Another approval for after pushing that commit may be needed to finally merge it to master.
```

As mentioned, will remove this with a follow up commit and merge